crates/bevy_core/src/time/time.rs

```diff
@@ -143,8 +155,16 @@ impl Time {
     }
 }
 
-pub(crate) fn time_system(mut time: ResMut<Time>) {
-    time.update();
+/// The system used to update the time. If there is a render world the time is sent from
+/// there to this system through channels. Otherwise the time is updated in this system.
+pub(crate) fn time_system(mut time: ResMut<Time>, time_recv: Option<Res<TimeReceiver>>) {
```

Comment:
If `TimeReciever` is always added in the plugin, this probably shouldn't be an `Option`.

Reply:
it's added in the render plugin. if users aren't using rendering then this makes the system still usable.